### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by NelsonVides**
Add github actions and circleci badges, correct a typo, and a link to the muc_light xep.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[9052.2](https://travis-ci.org/esl/MongooseIM/jobs/757081936) / Erlang 23.0.3 / internal_mnesia / 9303a4680438a3ffb3bd64215813dd1564de8f6a
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/3020/9052/internal_mnesia.23.0.3)/ [big](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/3020/9052/internal_mnesia.23.0.3/big/ct_run.test%40travis-job-b19df433-b9a8-40a6-b73e-6f20fd4b7067.2021-02-01_15.49.07/index.html)
**OK: 1503** / Failed: 0 / **User-skipped: 161** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[9053.1](https://travis-ci.org/esl/MongooseIM/jobs/757082599) / Erlang 23.0.3 / small_tests / 9015113c62a215ec254bceb27ea7dfa1119998b7
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/3020/9053/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/3020/9053/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2021-02-01_16.55.16/lib.mongooseim.logs/run.2021-02-01_16.55.16/suite.log.html)

File: doc/index.md [hunk 1]
```diff
@@ -65,11 +66,8 @@ For a quick start just download:
 
 Check out our test results:
 
-* Continuous integration: [https://travis-ci.org/esl/MongooseIM](https://travis-ci.org/esl/MongooseIM)
+* Continuous integration: [Github's actions](https://github.com/esl/MongooseIM/actions?query=workflow%3ACI)
```
Comment (leszke) [inline doc/index.md:69]: Isn't it called just `GitHub actions`? 🙂

File: README.md [hunk 2]
```diff
@@ -95,4 +92,4 @@ Suggestions, questions, thoughts? Contact us directly:
 * Email us at <a href='mailto:mongoose-im@erlang-solutions.com'>mongoose-im@erlang-solutions.com</a>
 * Follow our [Twitter account](https://twitter.com/MongooseIM): [https://twitter.com/MongooseIM](https://twitter.com/MongooseIM)
 * Like our [Facebook page](https://www.facebook.com/MongooseIM/): [https://www.facebook.com/MongooseIM/](https://www.facebook.com/MongooseIM/)
-* Subscribe to our [mailing list](https://groups.google.com/d/forum/mongooseim-announce) at [https://groups.google.com/d/forum/mongooseim-announce](https://groups.google.com/d/forum/mongooseim-announce) to receive no more than two montly emails as well as access to the free and open archives.
+* Subscribe to our [mailing list](https://groups.google.com/d/forum/mongooseim-announce) at [https://groups.google.com/d/forum/mongooseim-announce](https://groups.google.com/d/forum/mongooseim-announce) to receive no more than two monthly emails as well as access to the free and open archives.
```
Comment (leszke) [inline README.md:95]: Good catch 👌

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # MongooseIM platform
 
-[![Build Status](https://travis-ci.org/esl/MongooseIM.svg?branch=master)](https://travis-ci.org/esl/MongooseIM)
+[![Build Status](https://github.com/esl/MongooseIM/workflows/CI/badge.svg)](https://github.com/esl/MongooseIM/actions)
+[![CircleCI](https://circleci.com/gh/esl/MongooseIM.svg?style=shield&circle-token=53cdda11cfb95bd7a7044c509c89cb73d3fdeb5a)](https://github.com/esl/MongooseIM/)
```
Comment (leszke) [inline README.md:4]: Shouldn't it link to our circleci page? The same applies to `doc/index.md`.

File: doc/index.md [hunk 4]
```diff
@@ -98,4 +96,4 @@ Suggestions, questions, thoughts? Contact us directly:
 * Email us at <a href='mailto:mongoose-im@erlang-solutions.com'>mongoose-im@erlang-solutions.com</a>
 * Follow our [Twitter account](https://twitter.com/MongooseIM): [https://twitter.com/MongooseIM](https://twitter.com/MongooseIM)
 * Like our [Facebook page](https://www.facebook.com/MongooseIM/): [https://www.facebook.com/MongooseIM/](https://www.facebook.com/MongooseIM/)
-* Subscribe to our [mailing list](https://groups.google.com/d/forum/mongooseim-announce) at [https://groups.google.com/d/forum/mongooseim-announce](https://groups.google.com/d/forum/mongooseim-announce) to receive no more than two montly emails as well as access to the free and open archives.
+* Subscribe to our [mailing list](https://groups.google.com/d/forum/mongooseim-announce) at [https://groups.google.com/d/forum/mongooseim-announce](https://groups.google.com/d/forum/mongooseim-announce) to receive no more than two monthly emails as well as access to the free and open archives.
```
Comment (leszke) [inline doc/index.md:99]: I know it wasn't written by you and maybe I am nitpicking, but wouldn't it look better if we removed this links redundancy? Same for `README.md`.

**Review by leszke — COMMENTED** [review]
Thanks for this 😀 I left some comments below.

File: README.md (hunk 3 above)
Comment (NelsonVides) [reply README.md:4]: The circleci page is private, we're paying for a private account and only we have access. I've created a token within CircleCI (`token=53cdda11cfb95bd7a7044c509c89cb73d3fdeb5a` in the url) that has permissions only to see the status.

**Comment by mongoose-im** [comment]
[9061.1](https://travis-ci.org/esl/MongooseIM/jobs/757206772) / Erlang 23.0.3 / small_tests / c103048bc7c4696b174c1c7bb6a82ef122f7a277
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/3020/9061/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/3020/9061/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2021-02-02_09.02.28/lib.mongooseim.logs/run.2021-02-02_09.02.28/suite.log.html)

**Review by leszke — APPROVED** [review]
🚀 💪

**Review by chrzaszcz — COMMENTED** [review]
All good! I found an unrelated issue in the description of `mod_mam`:

https://github.com/esl/MongooseIM/blame/master/doc/modules/mod_mam.md#L66

I thin it should be moved to the description of the `host` option. Would it be possible to fix it in this PR?

**Comment by mongoose-im** [comment]
[9063.1](https://travis-ci.org/esl/MongooseIM/jobs/757223504) / Erlang 23.0.3 / small_tests / a49834d3f72559a305ad3c21b71005cddeae0751
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/3020/9063/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/3020/9063/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2021-02-02_13.16.23/lib.mongooseim.logs/run.2021-02-02_13.16.23/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Good!

**Pull request merged**